### PR TITLE
Remove plankton

### PR DESCRIPTION
**Pull request by manuelsc**

### CODE REVIEW

**Review by D13ce — DISMISSED** [review]
utACK

**Review by D13ce — APPROVED** [review]
Still utACK

File: src/app/utils/MerchantUtils.ts [hunk 1]
```diff
@@ -367,6 +370,13 @@ export class MerchantUtils {
 				return current
 			}
 		}
+		for (let i = 0; i < this.DEPRECATED_PACKAGES.length; i++) {
+			const current = this.DEPRECATED_PACKAGES[i]
+			if (current.purchaseKey == name) {
+				return current
```
Comment (beage666) [inline src/app/utils/MerchantUtils.ts:376]: []()true

Comment (beage666) [reply src/app/utils/MerchantUtils.ts:376]: True

**Review by beage666 — COMMENTED** [review]
True

**Pull request merged**